universal-application-tool-0.0.1/app/models/Application.java

```diff
@@ -47,6 +47,13 @@ public Application(Applicant applicant, Program program, LifecycleStage lifecycl
     this.lifecycleStage = lifecycleStage;
   }
 
+  public static Application createSync(
```

Comment:
The Sync name is not intuitive to me here.  Does is have inherent meaning in the Model semantics?

Reply:
Synchronous vs asynchronous DB interaction is distinction found throughout the codebase. By including "sync" in the method name I'm hoping to emphasize that it's a synchronous action, though I suppose that should be obvious since it doesn't return a promise. Thoughts on just naming it `create`?